### DISCUSSION TITLE
local machine -> own website

### DISCUSSION THREAD

**Discussion by sehrwahrscheinlich** [jas3333/GPT-Helper#11] [Q&A]
Hello all, 
I am new to github and unfortunately have very little idea about programming. I would like to fine-tune a chatbot and then make it accessible through my website. Can I customize this chatbot as a starting point? Is this legal and technically possible? Does anyone have any tips on where I can find more info that may be useful to me along the way?
Many thanks!

**Comment by jas3333** [comment]
As far as using the code from this chatbot you can do whatever you want with it. You would need to move all the API calls to a backend server(node/express, django, flask) to keep your API key from being exposed though. Also keep in mind if you have it publicly available you would quickly burn through tokens which could cost a lot of money.  If you're only giving access to certain people, you would need to setup some sort of authentication. I would check the offical docs for more info, they give an overview of what it would look like to setup. [OpenAI Docs](https://beta.openai.com/docs/introduction/overview). 

If you have any more questions I'll try my best to answer. I'm still learning a lot about this as well.

**Reply by sehrwahrscheinlich** [reply]
Wow, thanks a lot for your answer. That's a lot of work to do. I ll do my best :)
